azerothcore/azerothcore-wotlk
# fix(Script/ZulGurub) - High Priestess Jeklik improvements

## Changes Proposed:
- Add missing spells
- Fix fly issue in P2
- Fix Chain Mind Flay
- Fix Psychic Scream issue
- Adjust timers spells
- Add missing emotes
- Add missing aura before the fight 

## Issues Addressed:

- Closes #11381
- Closes #11382
- Closes a part of #11386
- Closes #11390 
- Closes #11380

## SOURCE:

Missing emotes, flying fix in P2, and few timers pick from https://github.com/TrinityCore/TrinityCore/pull/27274
(Thanks offl for you PR)

## Extra Notes:
After severals researches (TC, MangosClassic, other projects and issues...). It seems that nobody have found the right spell IDs from 
Chain mind flee. Also, this spells is rarely mentionned (maybe not exist on retail). So I deleted this spell for just let "Mind flay"

## Tests Performed:

- Tested IG

## How to Test the Changes:


1. .tele zulgurub
2. Engage Jeklik
3. Check that all spells are cast
4. Check some potential issues (except with bomb in P2, know bug)

## Known Issues and TODO List:


- [X] Timers need adjustements


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
